beacon_node/beacon_chain/src/schema_change.rs

```diff
@@ -73,6 +76,23 @@ pub fn migrate_schema<T: BeaconChainTypes>(
 
             Ok(())
         }
+        // Migration for weak subjectivity sync support and clean up of `OnDiskStoreConfig` (#1784).
```

Comment:
The migration only makes changes related to #1784 but the schema version has to change because a WSS database adds the `AnchorInfo` which an earlier version would be oblivious to (to its peril).